modules/ga.ts

```diff
@@ -2,6 +2,7 @@ export default function ga(...args) {
   if ("ga" in window) {
     window.ga(...args);
   } else {
+    // eslint-disable-next-line no-console
```

Comment:
The **console** statement was causing a `warning  Unexpected console statement  no-console` when executing `yarn checks` so added this quick fix here.